package.json

```diff
@@ -38,9 +37,11 @@
     "@jupyterlab/docregistry": "^0.11.0",
     "@jupyterlab/fileeditor": "^0.11.0",
     "@jupyterlab/services": "^0.50.0",
-    "@phosphor/widgets": "^1.3.0"
+    "@phosphor/widgets": "^1.3.0",
+    "pdfjs-dist": "^2.0.104"
   },
   "devDependencies": {
+    "@types/pdfjs-dist": "^0.1.0",
```

Comment:
what does this do? What is the @types scope doing?

Reply:
The `@types` namespace is a repository for community-contributed Typescript typings (i.e., type annotations for a Javascript package). So here we are importing both PDFJS and the separate typings for it.